pkg/logs/input/docker/tailer.go

```diff
@@ -175,14 +176,13 @@ func (t *Tailer) forwardMessages() {
 		t.done <- struct{}{}
 	}()
 	for output := range t.decoder.OutputChan {
-		if len(output.Content) > 0 {
```

Comment:
Is it safe to drop this check ? We don't want to send empty logs.

Reply:
It's safe, it's just a cleanup, we moved this check in the decoder, see https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/decoder/line_handler.go#L108

Reply:
cc @iksaif

Reply:
👍